lib/autocomplete-manager.coffee

```diff
@@ -98,6 +99,11 @@ class AutocompleteManager
     @subscriptions.add(atom.config.observe('autosave.enabled', (value) => @autosaveEnabled = value))
     @subscriptions.add(atom.config.observe('autocomplete-plus.backspaceTriggersAutocomplete', (value) => @backspaceTriggersAutocomplete = value))
     @subscriptions.add(atom.config.observe('autocomplete-plus.enableAutoActivation', (value) => @autoActivationEnabled = value))
+    @subscriptions.add atom.config.observe 'autocomplete-plus.suppressActivationForEditorClasses', (value) =>
+      unless value?.length
+        @suppressForClasses = []
+        return
+      @suppressForClasses = _.chain(value).reject((s) -> s.trim() is '').map((s) -> s.trim()).map((s) -> s.split('.')).value()
```

Comment:
k. You dont need that 3 lines above it. The `value` does not need to be checked for null as it is checked by config by way of the `array` type.